.gitignore

```diff
@@ -13,6 +13,5 @@ src/history/vacancy_history/data/cleaned
 src/history/vacancy_history/data/raw
 src/web/static/pages
 
-
-
 .DS_store
+style-errors.txt
```

Comment:
You shouldn't include this in `.gitignore`. There is nothing in the source code that generates this text file, and leaves me confused. If this is a temporary text file which was used when you were piping the output of `ruff check` into, then do not include it when you use `git add` to stage it.

Reply:
Oke noted. For reference I run the checkstyle by doing
`python -m ruff check > style-errors.txt`